server/app.js

```diff
@@ -28,6 +28,54 @@ app.use('/nasa', nasaRouter);
 //   next(createError(404));
 // });
 
+
```

Comment:
label code with // TODO: if code will be used later or remove if it is no longer needed